### PR TITLE
Add PSR7 and PSR17 aliases

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

This is a follow up on #546. I forgot to add some aliases.

### CODE REVIEW

File: nyholm/psr7/1.0/manifest.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "aliases": ["psr7", "psr-7","psr17", "psr-17"],
```
Comment (fabpot) [inline nyholm/psr7/1.0/manifest.json]: space missing after the second comma

Comment (Nyholm) [reply nyholm/psr7/1.0/manifest.json]: Thank you

**Comment by Nyholm** [comment]
> I thought about this also, but wouldn't one expect to get the bridge when doing "composer require psr7"?

Actually no =)
I would not expect the bridge when I just want a PSR7 implementation.. 

Suggestion: I modify this to alias "PSR-17", then create a pack for alias "PSR-7". That pack would include the bridge and nyholm/psr7.

**Comment by nicolas-grekas** [comment]
> That pack would include the bridge and nyholm/psr7

LGTM - we'd also need sensio-framework-extra-bundle in that pack so that everything works seamlessly, isn't it?

**Comment by Nyholm** [comment]
Correct

**Comment by Tobion** [comment]
Why should we alias this package for psr-7 for example? There are alot of other implementations that would qualify for this as well.

**Comment by fabpot** [comment]
Aliases are choices made by Symfony. This is a perfect example and does not prevent anyone from installing something else.

**Comment by Tobion** [comment]
To me, psr-7 should be added to https://github.com/symfony/psr-http-message-bridge or https://github.com/guzzle/psr7

**Pull request merged**